### DISCUSSION TITLE
discussion

### DISCUSSION THREAD

**Discussion by kmotaleb** [github-linguist/linguist#5223] [General]
Reporting a security vulnerability?

**Comment by lildude** [comment]
In Linguist?

**Reply by lildude** [reply]
To be clearer, I mean specifically are you referring to a vulnerability in Linguist's code under the `lib` directory and not one of fils under `samples`?

If you've found an issue in one of the files in `samples`, we know about a few files that have been flagged by GitHub's security scanner, but it's not an issue as those samples aren't actually used. They're tokenised for use by the classifier when the gem is created so doesn't represent usable code when used.
